sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/RxGatewayStoreModel.java

```diff
@@ -109,6 +109,10 @@ public RxGatewayStoreModel(
         return this.performRequest(request, HttpMethod.DELETE);
     }
 
+    private Mono<RxDocumentServiceResponse> deleteByPartitionKey(RxDocumentServiceRequest request) {
+        return this.performRequest(request, HttpMethod.POST);
```

Comment:
Is this need to be post ?

Reply:
Yes, according to the design doc, it has to go as POST.